app/javascript/components/App.jsx

```diff
@@ -21,14 +21,19 @@ class App extends Component {
       isLoggedIn: false,
       user: {},
       loading: true,
-      initialLoading: true
+      initialLoading: true,
+      vrsComponents: []
     };
   }
 
   componentDidMount() {
     this.getLoginStatus();
   }
 
+  selectVrsComponents = (vrsComponents) => {
```

Comment:
Naming of this function quite confusing. People might misinterpret it as a function to select the hrs compoenents. Can change to `selectedVrsComponents`